CHANGELOG.md

```diff
@@ -104,6 +119,21 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - Fix the scrolling issue
 
-## [0.1] 16/04/2023
+## [0.1] - 2023-04-16
 
 First release 🎉
+
+
+[unreleased]: https://github.com/WyvernIXTL/ubilerntui/compare/v0.11.1...HEAD
+[0.11.1]: https://github.com/pythops/tenere/compare/v0.11...v0.11.1
+[0.11]: https://github.com/pythops/tenere/compare/v0.10...v0.11
+[0.10]: https://github.com/pythops/tenere/compare/v0.9...v0.10
+[0.9]: https://github.com/pythops/tenere/compare/v0.8...v0.9
+[0.8]: https://github.com/pythops/tenere/compare/v0.7...v0.8
+[0.7]: https://github.com/pythops/tenere/compare/v0.6...v0.7
+[0.6]: https://github.com/pythops/tenere/compare/v0.5...v0.6
+[0.5]: https://github.com/pythops/tenere/compare/v0.4...v0.5
+[0.4]: https://github.com/pythops/tenere/compare/v0.3...v0.4
+[0.3]: https://github.com/pythops/tenere/compare/v0.2...v0.3
+[0.2]: https://github.com/pythops/tenere/compare/v0.1...v0.2
+[0.1]: https://github.com/pythops/tenere/releases/tag/v0.1
```

Comment:
why is that ?

Reply:
Please have a look at keepachangelogs website. Brackets denote a link
 That's why the versions are in bracket. So we can link the changes between them.